apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -60,6 +60,27 @@ function is_homepage_title_hidden() {
 	return (bool) is_block_editor_screen() && $hide_homepage_title && $is_homepage;
 }
 
+/**
+ * Detects if the site is using Gutenberg 9.2 or above, which contains a bug in the
+ * interface package, causing some "slider" blocks (such as Jetpack's Slideshow) to
+ * incorrectly calculate their width as 33554400px when set at full width.
+ *
+ * @see https://github.com/WordPress/gutenberg/pull/26552
+ *
+ * @return bool True if the site needs a temporary fix for the incorrect slider width.
+ */
+function needs_slider_width_workaround() {
+	$gutenberg_path = ABSPATH . 'wp-content/plugins/gutenberg/gutenberg.php';
```

Comment:
We can probably check if `GUTENBERG_VERSION` [is defined](https://plugins.trac.wordpress.org/browser/gutenberg/tags/9.2.2/gutenberg.php#L16) and read the value.

This path likely won't work well on WPCOM.

Reply:
I did try `GUTENBERG_VERSION` on my local install, and it returns undefined. 🤔

Reply:
Oh, I think that constant is only generated upon Gutenberg release.
The development version instead has just a `define( 'GUTENBERG_DEVELOPMENT_MODE', true );`.

I guess using **both** `GUTENBERG_VERSION` **and** `GUTENBERG_DEVELOPMENT_MODE` should make this safe enough.